MangoAPI.BusinessLogic/DependencyInjection/AzureBlobInitializer.cs

```diff
@@ -15,6 +18,16 @@ public static void InitializeAzureBlob(this IApplicationBuilder app)
 
         var blobService = serviceScope.ServiceProvider.GetService<IBlobService>();
 
+        var runningInContainerEnvValue = Environment.GetEnvironmentVariable(EnvironmentConstants.AspNetCoreRunningInContainer);
```

Comment:
Instead of changing folder path -> you can change path images copied to so that it will match `../../../../img/seed_images`

Reply:
Honestly, I don't see anything wrong with it. Container file system with your approach would seem very bad